containers/ubuntu-lts-builder/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt install -y \
 RUN apt install -y openjdk-11-jdk-headless openjdk-17-jdk-headless
 
 # Buildbot worker dependencies
-RUN apt install -y ninja-build buildbot-worker clang-format-12 clang-format-13
+RUN apt install -y ninja-build buildbot-worker clang-format-12 clang-format-13 clang-format-15
```

Comment:
Do we need the old versions? The previous change on this line added 13 without removing 12, but I wonder if that's because the Ubuntu builder at the time was on an older version that didn't come with 13 by default.

Reply:
I was following OatmealDome's advice that I "probably should just add the new version onto the end so all three versions are installed". I don't know what benefit this might have.